Assets/Scripts/Models/Prototypes/PrototypeManager.cs

```diff
@@ -20,6 +21,7 @@ public PrototypeManager()
         Need = new NeedPrototypes();
         Trader = new TraderPrototypes();
         Quest = new QuestPrototypes();
+        Event = new EventPrototypes();
```

Comment:
Should we call this SchedulerEvent and SchedulerEventPrototypes, since there are other events (GameEvents) and there can be more in the future.